README.md

```diff
@@ -1,12 +1,13 @@
 # IOTA-Balanace-Checker
 
 ### Description:
-A simpel python script to generate the first X addresses of a seed and check there balances
+A simple python script to generate the first X addresses of a seed and check there balances
 
 
 ### Usage:
 
-Note: You need the PyOTA library installed for the script to work. You can install PyOTA with `pip install pyota`
+Note: You need the PyOTA and six libraries installed for the script to work.
+You can install them with `pip install -r requirements.txt`.
```

Comment:
A common convention when releasing code that has dependencies is to include a `requirements.txt` file.  This makes it easy for other developers to install the project's dependencies with a single command (: